cmd/kots/cli/ingress.go

```diff
@@ -91,9 +80,18 @@ func IngressInstallCmd() *cobra.Command {
 
 			log.FinishSpinner()
 
+			identityConfig, err := identity.GetConfig(cmd.Context(), namespace)
+			if err != nil {
+				return errors.Wrap(err, "failed to get identity config")
+			}
+
 			if identityConfig.Spec.Enabled {
 				log.ChildActionWithSpinner("Configuring the Identity Service")
 
+				if err := identity.ConfigValidate(identityConfig.Spec, ingressConfig.Spec); err != nil {
```

Comment:
shouldnt config be validated before it is set and you have an invalid config?

Reply:
this is the identity config, and i don't see it being set anywhere

Reply:
but it validates that your ingress config is valid for your identity config

https://github.com/replicatedhq/kots/pull/1299/files#diff-4675db0230b0c53e97f1cf8e5fdbb0d03a22ebda68410213fb184cc8a17095c6R73